src/styles/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  gap: $cells-gap;
+  max-width: $calendar-max-width;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+
+  margin: 50px auto;
+
+  &__day {
+    @include square($cell-size);
+
+    background-color: #eee;
+    border: 1px solid black;
+    box-sizing: border-box;
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    cursor: pointer;
+    transition: background-color 0.5s, transform 0.5s;
+
+    @for $day from 1 through 31 {
+      &:nth-child(#{$day}):before {
+        content: "#{$day}";
+      }
+    }
+  }
+
+  &__day:hover {
+    background-color: rgb(255, 107, 187);
+    transform: translate(0, -20px);
+  }
+
+  @for $last-day from 28 through 31 {
+    &--month-length_#{$last-day}
+    &__day:nth-child(n + #{$last-day + 1}) {
+      display: none;
+    }
+  }
+
+  @each $name, $num in $calendar-day {
```

Comment:
Maybe, it's better to create another file for @for and @each loop, but I'm not sure if it's necessary for your code

Reply:
No, keep them here.